java/runtime-native/src/main/java/org/ray/spi/impl/JarLoader.java

```diff
@@ -0,0 +1,88 @@
+package org.ray.spi.impl;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Enumeration;
+import java.util.List;
+import java.util.jar.JarEntry;
+import java.util.jar.JarFile;
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.IOUtils;
+import org.apache.commons.io.filefilter.DirectoryFileFilter;
+import org.apache.commons.io.filefilter.RegexFileFilter;
+import org.ray.util.logger.RayLog;
+
+/**
+ * load and unload jars from a dir.
+ */
+public class JarLoader {
+
+  public static URLClassLoader loadJars(String dir, boolean explicitLoad) {
+    // get all jars
+    Collection<File> jars = FileUtils.listFiles(
+        new File(dir),
+        new RegexFileFilter(".*\\.jar"),
+        DirectoryFileFilter.DIRECTORY
+    );
+    return loadJar(jars, explicitLoad);
+  }
+
+  public static void unloadJars(ClassLoader loader) {
+    // TODO:
```

Comment:
say what is to be done here (or if nothing, remove the todo)

Reply:
ok, I would add the doc